### ISSUE TITLE
flutter logs output does not match flutter run output

### ISSUE THREAD

**Issue by safield** [flutter/flutter#70846]
Flutter version 1.22.2
Running on Android Physical Device and Android Emulator.

I have noticed that using _flutter run_ results in different output from _flutter logs_ command.

The expected behaviour is that log output captured from the device would match in both _flutter run_ and _flutter logs_

Here is a comparison of the same scenario on the device, producing different logs captured from the device.

**flutter logs**
```
Showing Moto G 5 Plus logs:

I/FlutterBluePlugin(11708): setup
I/flutter (11708): Observatory listening on http://127.0.0.1:41088/ges8GJXaKYU=/

```
**flutter run**
```
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on Moto G 5 Plus is available at: http://127.0.0.1:46703/ewlO1sM0cuY=/
W/ConnectionTracker(14181): Exception thrown while unbinding
W/ConnectionTracker(14181): java.lang.IllegalArgumentException: Service not registered: lv@417ba8c
W/ConnectionTracker(14181):     at android.app.LoadedApk.forgetServiceDispatcher(LoadedApk.java:1493)
W/ConnectionTracker(14181):     at android.app.ContextImpl.unbindService(ContextImpl.java:1653)
W/ConnectionTracker(14181):     at android.content.ContextWrapper.unbindService(ContextWrapper.java:703)
W/ConnectionTracker(14181):     at ci.f(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):1)
W/ConnectionTracker(14181):     at ci.d(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):2)
W/ConnectionTracker(14181):     at lw.E(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):9)
W/ConnectionTracker(14181):     at lg.a(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):3)
W/ConnectionTracker(14181):     at eg.run(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):3)
W/ConnectionTracker(14181):     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:457)
W/ConnectionTracker(14181):     at java.util.concurrent.FutureTask.run(FutureTask.java:266)
W/ConnectionTracker(14181):     at iz.run(:com.google.android.gms.dynamite_measurementdynamite@204514047@20.45.14 (040308-0):5)
```

Flutter run has captured a large amount of warnings from the device that flutter logs did not capture.

**Comment by TahaTesser** [comment]
Hi @safield 
Where you getting `flutter logs`? Please provide steps to reproduce the issue and does it reproduce it on the emulator as well?
Can you please provide your `flutter doctor -v` and `flutter run --verbose` logs, and a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?
Thank you

**Comment by safield** [comment]
flutter logs is a flutter command...lol

flutter run --verbose is not relevant here as the issue is literally related to the output of flutter run. There is no minimal complete code sample here, as you just have to run flutter run and flutter logs in parallel and observe their output.

Your response put about as much critical assessment into the issue as a bot...

**Comment by darshankawar** [comment]
Hi @safield 
Tried both commands on latest stable (1.22.4) and noticed similar output using Android device (8.1.0) and an emulator (Pixel 3XL)

<img width="956" alt="Screenshot 2020-11-24 at 3 46 30 PM" src="https://user-images.githubusercontent.com/67046386/100081318-e481f580-2e6c-11eb-9de0-564e6f63dd7e.png">
<img width="1085" alt="Screenshot 2020-11-24 at 3 47 19 PM" src="https://user-images.githubusercontent.com/67046386/100081333-e77ce600-2e6c-11eb-83f8-c74d67b96ed1.png">

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 1.22.4, on Mac OS X 10.15.4 19E2269 darwin-x64,
    locale en-IN)
    • Flutter version 1.22.4 at /Users/dhs/documents/Fluttersdk/flutter
    • Framework revision 1aafb3a8b9 (11 days ago), 2020-11-13 09:59:28 -0800
    • Engine revision 2c956a31c0
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /Users/dhs/Library/Android/sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0.1, Build version 12A7300
    • CocoaPods version 1.9.3

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.50.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.15.1

[✓] Connected device (3 available)
    • SM A260G (mobile)                   • 5200763ebcfa861f
      • android-arm • Android 8.1.0 (API 27)
    • sdk gphone x86 arm (mobile)         • emulator-5554
      • android-x86 • Android 11 (API 30) (emulator)
    • iPhone SE (2nd generation) (mobile) • 6C85835D-FBFD-4AB3-8DE8-B4FAD35E5367
      • ios         • com.apple.CoreSimulator.SimRuntime.iOS-14-0 (simulator)

• No issues found!


```

</details>

Can you upgrade to latest stable and try again to see if issue still persists ?
Thanks.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
